packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -385,4 +385,24 @@ module.exports = ({ strapi }) => ({
 
     return sumDraftCounts(entity, uid);
   },
+
+  async getMultipleEntriesNumberOfDraftRelations(ids, uid) {
```

Comment:
Don't know if it's only me, but I always felt like `getNumberOfDraftRelations` name was off, or at least it was not consistent on how we name methods, still it's clear what its intent is.

I was trying to thing of alternatives and thought of:

<table>
<tr>
 <td> Before
 <td>After
<tr>
 <td>getNumberOfDraftRelations
 <td>countDraftRelations
<tr>
 <td>getMultipleEntriesNumberOfDraftRelations
 <td>countManyEntriesDraftRelations
</table>


wdyt?

Reply:
I agree, I was trying to think of some different names earlier as well but came up short. I like your suggestions 👍

Reply:
I was not sure if countOneDraftRelations / countDraftRelations was better naming, but I think it's not as clear as the other ones

Reply:
I think I prefer you first suggestion. But this does conform more with the findOne / find approach already in the service.

Another option, but more verbose: countDraftRelationsForEntity / countDraftRelationsForEntities

Reply:
I agree with your suggestions, I changed the name of the two methods as recommended by Marc